appveyor.yml

```diff
@@ -9,28 +9,38 @@ environment:
   matrix:
 
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-      TOOLCHAIN: "ninja-vs-15-2017-win64-cxx17"
-      PROJECT_DIR: examples\foo
-
-    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-      TOOLCHAIN: "nmake-vs-15-2017-win64-cxx17"
-      PROJECT_DIR: examples\foo
-
-    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-      TOOLCHAIN: "vs-15-2017-win64-cxx17"
-      PROJECT_DIR: examples\foo
-
-    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-      TOOLCHAIN: "vs-14-2015-sdk-8-1"
-      PROJECT_DIR: examples\foo
-
-    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-      TOOLCHAIN: "mingw-cxx17"
-      PROJECT_DIR: examples\foo
-
-    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-      TOOLCHAIN: "msys-cxx17"
-      PROJECT_DIR: examples\foo
+      TOOLCHAIN: "dummy"
```

Comment:
Please add `dummy` after the main table and put it into `# Extra {`/`# }` block. See example:
* https://github.com/ingenue/hunter/blob/479d4dafad450f6ea0024934a529021bfbaf3c49/appveyor.yml#L10-L48